### ISSUE TITLE
How is the dataset divided?

### ISSUE THREAD

**Issue by liuwenjun-test** [tianrun-chen/xLSTM-UNet-PyTorch#20]
The abdominal organ dataset consists of a training set and a test set. So, how is validation conducted? Is it five-fold cross-validation? Is it still 8:2? Has there been no division yet? Will the testing be conducted directly after the training is completed?

 Does anyone have a script for evaluating the BRATS data?
